### PR TITLE
colibread tools: allow gz input

### PR DESCRIPTION
**Pull request by gregvonkuster**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/colibread/takeabreak.xml [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
-
-        ## if single reads
-        #if str( $input_type_option.input_type ) == "simple"
+    <command detect_errors="exit_code"><![CDATA[
+        #import re
+        #if str($input_type_option.input_type) == "simple"
             #for $input in $input_type_option.reads
-                #if $input
-                    #set $filename = os.path.basename(str($input)) + "." + $input.ext
+                #set $base_filename = re.sub('[^\w_]', '_', $input.element_identifier)
```
Comment (bgruening) [inline tools/colibread/takeabreak.xml:13]: @gregvonkuster should we reuse this in a macro, its now used a lot all over the place.

**Review by bgruening — COMMENTED** [review]
Great clean up! The only question I have is if macros should be used more and if we should bump the profile version.

File: tools/colibread/takeabreak.xml (hunk 1 above)
Comment (gregvonkuster) [reply tools/colibread/takeabreak.xml:13]: Yes, I'll do this.

**Comment by gregvonkuster** [comment]
What profile version should I bump it to?

**Comment by bgruening** [comment]
20.01 I think.

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**